k9mail/src/main/java/com/fsck/k9/activity/MessageCompose.java

```diff
@@ -235,6 +235,7 @@
     private RecipientPresenter recipientPresenter;
     private MessageBuilder currentMessageBuilder;
     private boolean mFinishAfterDraftSaved;
+    private boolean mFirstTimeEmptySubject = true;
```

Comment:
I don't like the name because it doesn't really convey what it's all about. I think something along the lines of `alreadyNotifiedUserOfEmptySubject` works better (we don't use the "m" prefix for new fields anymore).